### DISCUSSION TITLE
Problem creating graph in Rstudio

### DISCUSSION THREAD

**Discussion by alexmcd00** [dat-linux/community#25] [FAQ] [answered]
Hey! I am learning to use Rstudio for an assignment.
I am trying to create a basic graph using data which I imported from Excel. 
Can anyone tell me what I am doing wrong? 

<img width="694" alt="Screenshot 2023-04-07 at 3 21 39 pm" src="https://user-images.githubusercontent.com/130126998/230546277-39afe08a-3135-4326-bedf-989cc113f55b.png">

**Comment by dat-linux** [comment, marked as answer]
Is **Fertility_rate** spelled correctly? Look at the dataset to see. It is case sensitive. Attribute name must be exact match. Run:

`View(FR)`

or 

`head(FR)`
